geo/src/algorithm/offset/offset_trait.rs

```diff
@@ -0,0 +1,336 @@
+use super::line_intersection::FalseIntersectionPointType::{
+    BeforeStart,
+    AfterEnd,
+};
+use super::line_intersection::LineSegmentIntersectionType::{
+    FalseIntersectionPoint, TrueIntersectionPoint,
+};
+use super::line_intersection::{
+    line_segment_intersection_with_relationships, LineIntersectionWithParameterResult,
+};
+use super::slice_itertools::pairwise;
+
+use crate::{
+    Coord,
+    CoordFloat,
+    // Kernel,
+    // Orientation,
+    Line,
+    LineString,
+    MultiLineString,
+    // Polygon,
+};
+
+/// # Offset Trait
+///
+/// Signed offset of Geometry assuming cartesian coordinate system.
+///
+/// This is a cheap offset algorithm that is suitable for flat coordinate systems

```

Comment:
There's no need to specify this as `geo` operates only on coordinates in the real plane (with the exception of some explicitly documented convenience algorithms for spherical coordinates). There is an explicit assumption that only projected / cartesian coordinates are used. That this algorithm may be suitable for coordinates near the equator is a coincidence as far as the library is concerned.